symplyphysics/laws/thermodynamics/first_law_of_termodynamics.py

```diff
@@ -0,0 +1,35 @@
+from sympy import (Eq, solve)
```

Comment:
We have a convention of not using "marketing" law names, like "Newton's first law", but explanatory names, eg "some_value_from_something". Could you rename the law according to this rule?